cam2/src/main/java/com/commonsware/cwac/cam2/ConfirmationFragment.java

```diff
@@ -49,6 +49,7 @@
   private ImageHelper imageHelper = new ImageHelper();
   private boolean facePass = false;
   private boolean sensorPass = false;
+  private boolean faceClosePass = false;
```

Comment:
other then initialisation facePass and faceClosePass is always made true, never assigned a false value